src/app/swap/swap-listener.ts

```diff
@@ -0,0 +1,68 @@
+import { toSats } from "@domain/bitcoin"
+
+import { admin as LedgerAdmin } from "@services/ledger/admin"
+import { WalletCurrency } from "@domain/shared"
+import { LedgerTransactionType } from "@domain/ledger"
+import { SwapProvider, SwapState, SwapType } from "@domain/swap"
+import { addAttributesToCurrentSpan } from "@services/tracing"
+import { SwapErrorNoActiveLoopdNode } from "@domain/swap/errors"
+import { sha256 } from "@domain/bitcoin/lightning"
+
+export const startSwapMonitor = async (swapService: ISwapService) => {
```

Comment:
Wondering if this function belongs in a `server` file.

Reply:
it gets called by the `trigger.ts` server